aggregator_api/src/lib.rs

```diff
@@ -435,12 +458,153 @@ async fn get_task_metrics<C: Clock>(
     }))
 }
 
+async fn get_global_hpke_keypairs<C: Clock>(
+    _: &mut Conn,
+    State(ds): State<Arc<Datastore<C>>>,
+) -> Result<impl Handler, Status> {
+    Ok(Json(
+        ds.run_tx_with_name("get_global_hpke_keypairs", |tx| {
+            Box::pin(async move { tx.get_global_hpke_keypairs().await })
+        })
+        .await
+        .map_err(|err| {
+            error!(err = %err, "Database transaction error");
+            Status::InternalServerError
+        })?
+        .into_iter()
+        .map(GlobalHpkeKeypairResp::from)
+        .collect::<Vec<_>>(),
+    ))
```

Comment:
I considered having a `get_global_hpke_keypair_config` function or thereabouts that doesn't load the private key into memory. While the private key has to be in memory for report decryption, this would lower the discrete locations in memory that it's kept. It also mitigates a class of bug where we respond (or are coerced to respond) with confidential information.

However, I'm not sure if it's worth the extra code/effort. Open to opinions on this.